dataflow/flex-templates/kafka_to_bigquery/metadata.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Ideally should have Copyright & License unless whatever reading it has a problem w/ that.
 "_Copyright" : "Copyright 2020 Google, LLC",
"_License":"....",
If that doesn't work, then it's ok to skip.